src/renderers/WebGLRenderer.js

```diff
@@ -262,6 +263,7 @@ THREE.WebGLRenderer = function ( parameters ) {
 	this.context = _gl;
 	this.extensions = extensions;
 	this.state = state;
+	this.logarithmicDepthBuffer = _logarithmicDepthBuffer;
```

Comment:
Instead of adding properties to the renderer I would pass this in the `THREE.WebGLProgramCache` constructor. It's a temporal ugliness, but that way we don't have issues in case users start accessing that property.

I suspect that's what you mean with `WebGLCapabilities`?